### PR TITLE
Updates caveats re: permission issues

### PR DESCRIPTION
**Pull request by rednoah**
Is there any way to change permissions after install? Maybe cask formulae support that? Maybe some post-install hook?

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Yes, what you are looking for is [postflight](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#postflight-stanza-details).

[Example](https://github.com/caskroom/homebrew-cask/blob/a0ebac74bf73f7b3570a68666c3eb46646abdb0d/Casks/onlabs.rb#L15-17).

**Comment by rednoah** [comment]
Added chown post-flight instructions

File: Casks/sencha.rb [hunk 1]
```diff
@@ -15,18 +15,27 @@
...
-      Sencha Cmd adds two changes to your ~/.bashrc file:
+      Sencha Cmd adds two changes to your ~/.bashrc or ~/.profile file:
 
         export PATH=/opt/Sencha/Cmd/#{version}:$PATH
         export SENCHA_CMD_3_0_0="/opt/Sencha/Cmd/#{version}"
 
-      If you are a zshell user, copy at the end of your .zshrc file both lines.
+      If you are a zshell user, append both lines to your .zshrc file.
+
+      Sencha Cmd requires read/write permissions to it's application folder:
```
Comment (vitorgalvao) [inline Casks/sencha.rb]: Doesn’t the postflight mean this instructions can be remove?

**Comment by rednoah** [comment]
Updated caveats.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by rednoah** [comment]
too complicated for an issue that can be solved with copy & paste

**Comment by vitorgalvao** [comment]
Could never understand why capable users throw in the towel when everything is done and a simple rebase is all that is needed.

Very well, then. Thank you for the contribution. Will be merged in https://github.com/caskroom/homebrew-cask/pull/11228 with squashed commits. Your contribution is still included (and still credited to you), with the appropriate modifications.

**Comment by vitorgalvao** [comment]
Just realised what you meant, thanks to #11226. I thought you were giving up on the PR altogether so far into it, which happens more times than one would expect.

Apologies for the assumption, and once more thank you for the PR.

**Pull request closed without merging**